katas/mumbling/README.md

```diff
@@ -0,0 +1,20 @@
+# Mumbling Kata
+
+The goal of this kata is to implement the `accum()` method which takes a string as input and returns a formatted output string.
```

Comment:
Might be worth expanding on the rules of the formatting.
You can deduce it from the example code you provide, but it's better to be explicit.

Reply:
I've now amended it to provide some clarity on the expected output